client/src/modules/privacy-policy/index.jsx

```diff
@@ -28,110 +29,93 @@ class PrivacyPolicy extends PureComponent {
       <Fragment>
         <div className="privacy-policy">
           <h1 className="privacy-policy__intro-heading">
-            Privacy <span>& Terms</span>
+            <FormattedMessage id="privacy-policy.index.heading" />
           </h1>
           <h2 className="privacy-policy__intro-subheading">
-            Check our Privacy Policy <span>and Terms.</span>
+            <FormattedMessage id="privacy-policy.index.subheading" />
           </h2>
           <div className="privacy-policy__info-wrapper">
             <article className="privacy-policy__info">
-              <h2 className="privacy-policy__header">Privacy Policy</h2>
+              <h2 className="privacy-policy__header">
+                <FormattedMessage id="privacy-policy.index.privacy-policy" />
+              </h2>
               <p className="privacy-policy__content">
-                Sandstream Development (herein referred to as “Sandstream”) has
-                created this Privacy Policy in line with our commitment to your
-                privacy on EOC Application (herein referred to &apos;App&apos;).
-                This app informs you of our policies regarding the collection,
-                use and disclosure of Personal Information, Information Security
-                and Quality, and Cookies.
+                <FormattedMessage id="privacy-policy.index.content-1" />
               </p>
             </article>
           </div>
           <div className="privacy-policy__info-wrapper">
             <article className="privacy-policy__info">
-              <h2 className="privacy-policy__header">Personal Information</h2>
+              <h2 className="privacy-policy__header">
+                <FormattedMessage id="privacy-policy.index.header-1" />
+              </h2>
               <p className="privacy-policy__content">
-                Our privacy practices are designed to provide a high level of
-                protection for your personal data. We use your Personal
-                information only for providing and improving the App. You agree
-                to the collection and use of this information in accordance with
-                this policy.
+                <FormattedMessage id="privacy-policy.index.content-2" />
```

Comment:
consider: to make this id more descriptive e.g. privacy-policy.index.content-information

Reply:
check above answer